app/bundles/PageBundle/Translations/en_US/messages.ini

```
@@ -182,6 +182,8 @@ mautic.config.tab.tracking.facebook.pixel="Facebook Pixel"
 mautic.page.config.form.facebook.pixel.id="Facebook Pixel ID"
 mautic.config.tab.tracking.google.analytics="Google Analytics"
 mautic.page.config.form.google.analytics.id="Google Analytics ID"
+mautic.config.tab.tracking.google.tagmanager="Google Tag Manager"
+mautic.page.config.form.google.tagmanager.id="Google Tag Manager ID"
 mautic.page.config.form.tracking.landingpage.enabled="Enable on Mautic landing page"
 mautic.page.config.form.tracking.trackingpage.enabled="Enabled on your tracking page"
```

I would maybe make this a bit clearer what we mean by 'tracking page' to make it clear that we mean pages external to Mautic where the tracking code is being used:

```suggestion
mautic.page.config.form.tracking.trackingpage.enabled="Enabled on your externally tracked pages"
```